internal/kibana/policies.go

```diff
@@ -193,12 +193,12 @@ type DataStream struct {
 
 // Input represents a package-level input.
 type Input struct {
-	Type           string        `json:"type"`
-	Enabled        bool          `json:"enabled"`
-	Streams        []interface{} `json:"streams"`
-	Vars           Vars          `json:"vars,omitempty"`
-	Config         interface{}   `json:"config,omitempty"`
-	CompiledStream interface{}   `json:"compiled_stream,omitempty"`
+	Type           string      `json:"type"`
+	Enabled        bool        `json:"enabled"`
+	Streams        []Stream    `json:"streams",omitempty`
```

Comment:
```suggestion
	Streams        []Stream    `json:"streams,omitempty"`
```

Reply:
Ah thanks for the fix and the recommendation. Will push the initialisation ASAP